src/components/Workspace.jsx

```diff
@@ -133,9 +138,88 @@ export default class Workspace extends React.Component {
     );
   }
 
+  _renderHiddenLanguages() {
+    const {
+      currentProject,
+      hiddenLanguages,
+      onComponentToggle,
+    } = this.props;
+    return (
+      <React.Fragment>
```

Comment:
I think we prefer the empty `<>` style for fragments (seems like there should be a linter rule for this?)